install.sh

```diff
@@ -54,6 +54,15 @@ Usage:
   --rsh <bin>
       Specifies the remote shell for remote installation. Defaults to ssh.
 
+  --install-terraform
```

Comment:
Since this script is called `install.sh`, what do you think about calling this `--with-terraform`?

Reply:
Or since it stops early and just installs Terraform, what about just `--terraform`?